app/assets/stylesheets/app/profile.scss

```diff
@@ -52,6 +52,11 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
+  
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Unexpected whitespace at end of line   no-eol-whitespace